docs/src/code/building-linuxcnc.adoc

```diff
@@ -5,34 +5,84 @@
 
 == Introduction
 
-This document describes how to build the LinuxCNC software and
-documentation from source.  This is primarily useful if you are a
-developer who is modifying LinuxCNC.  It can also be useful if you're
-a user who is testing developer branches, though then you also have
-the option of just installing Debian packages from the buildbot:
-http://buildbot.linuxcnc.org
+This document describes how to build the LinuxCNC software from source.
+This is primarily useful if you are a developer who is modifying LinuxCNC.
+It can also be useful if you're a user who is testing developer branches,
+though then you also have the option of just installing Debian packages from the buildbot
+(http://buildbot.linuxcnc.org) or as a regular package from your Linux
+distribution (https://tracker.debian.org/pkg/linuxcnc).
+Admittedly, this section also exists since LinuxCNC is a community effort.
+and you are encouraged to contribute to the development of LinuxCNC.
+You want to compile LinuxCNC yourself for The exchange of new developments,
+or you may be porting LinuxCNC to some new Linux disribution
+or developer reacts to you reporting a problem that you want to test
+but any such change will see no buildbot to help, or that help is delayed,
+pending someone else's review that you do not want to wait for or you are
```

Comment:
Typo?  else's -> elses

Reply:
No typo - see https://en.wikipedia.org/wiki/Somebody_else%27s_problem . Only "it's" is "its" :)